contrib/xdp-filter.ebpf.src

```diff
@@ -383,8 +425,10 @@ int xdp_dns_filter(struct xdp_md *ctx)
   		if (!(ipv4 = parse_iphdr(&c)) || bpf_htons(ipv4->protocol != IPPROTO_UDP)) {
   			return XDP_PASS;
   		}
+      struct CIDR4 key;
+      key.addr = bpf_htonl(ipv4->saddr);
       // if TC bit must not be set, apply the action
```

Comment:
Just a nit but the indentation is off here, as the comment was already misindented.

Reply:
I think it might be caused by this code mixing tabs and spaces. This code is indented correctly when it shows tabs as two spaces in my IDE.